manifests/image.pp

```diff
@@ -144,11 +144,19 @@
     notify { "Check if image ${image_arg} is in-sync":
       noop      => false,
     }
+    ~> exec { $image_install:
+      environment => $exec_environment,
+      path        => $exec_path,
+      timeout     => $exec_timeout,
+      returns     => ['0', '2'],
+      require     => File[$update_docker_image_path],
+      provider    => $exec_provider,
+      logoutput   => true,
```

Comment:
This probably needs the `refreshonly` metaparam set to true to ensure noop behaviour works as before.